packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AddToHomeScreenPrompt.tsx

```diff
@@ -9,6 +9,10 @@ interface AddToHomeScreenPromptProps {
   displayDelayMilliseconds?: number;
 }
 
+function isDev() {
+  return document.location.host === 'localhost:8080';
```

Comment:
we can visit the site on local IP ex: `127.0.0.1:8080` and still be in dev -- let's add a condition here to account for this